Sources/ArcGISToolkit/Components/FormView/FormInputs/DateTimeInput.swift

```diff
@@ -190,15 +181,6 @@ struct DateTimeInput: View {
         }
     }
     
-    /// The human-readable date and time selection.
-    var formattedDate: String? {
-        if input.includeTime {
-            return date?.formatted(.dateTime.day().month().year().hour().minute())
-        } else {
-            return date?.formatted(.dateTime.day().month().year())
-        }
-    }
```

Comment:
I'm seeing some new failures in our UI tests, specifically 2.3, 2.4 and 2.5. Should the formatted value from core respect `DateTimePickerFormInput.includeTime` for us now?

Before:
<img width="227" alt="Screenshot 2024-01-03 at 14 45 55" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/be414738-ef4d-463a-91db-adcc4590eb7d">

After:
<img width="232" alt="Screenshot 2024-01-03 at 14 49 24" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/295fdc0a-bd17-4605-908b-225beeee9268">

Reply:
I would think so, yes.